src/master-playlist-controller.js

```diff
@@ -698,7 +698,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.masterPlaylistLoader_.media(media);
 
       this.mainSegmentLoader_.resetLoader();
-      if (this.audiosegmentloader_) {
+      if (this.audioPlaylistLoader_) {
```

Comment:
audioSegmentLoader_ always exists, even if not being used. audioPlaylistLoader_ only exists if we are using alt audio